# Velodyne LIDAR status messages.

string  gps_position    #
string  gps_state       #
bool    motor_state     #
uint16  motor_rpm       #
bool    motor_lock      #
uint16  motor_phase     #
bool    laser_state     #
